src/components/inputs/CommentInput.tsx

```diff
@@ -27,13 +27,15 @@ export const CommentInput = ({ createComment, task_id }: Prop) => {
   const currentUserDetails = assignee.find((el) => el.id === currentUserId)
 
   const handleSubmit = () => {
-    const commentPayload: CreateComment = {
-      content: detail,
-      taskId: task_id,
-      mentions: getMentionsList(detail),
+    if (detail) {
+      const commentPayload: CreateComment = {
+        content: detail,
+        taskId: task_id,
+        mentions: getMentionsList(detail),
+      }
+      createComment(commentPayload)
+      setDetail('')
```

Comment:
Please check for all cases surrounding this feature as well.
- Can post comment with spaces only
- Can post comment with newline only
![image](https://github.com/user-attachments/assets/985fce8d-239a-4c26-8102-f899cc805c8d)

Reply:
Fixed in the commit below @rrojan